custom-connectors/secplugs/apiDefinition.swagger.json

```diff
@@ -0,0 +1,445 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Secplug Scan",
+    "description": "Secplug Scan for scanning Emails, files and Urls",
+    "version": "1.0",
+	"contact": {
+      "name": "Secplugs",
+      "url": "https://secplugs.com/",
+      "email": "secplugs@secplugs.com"
+    }	
+  },
+  "host": "proxy.secplugs.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/file/jsonupload": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "score": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "score"
+                },
+                "json_report": {
+                  "type": "array",
+                  "items": {},
+                  "description": "json_report"
```

Comment:
@Ajaatsecplugs Please provide valid description value as it seems like an operation ID.

Documentation - [Summary and Description Values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the description values